test/unit/spread.spec.ts

```diff
@@ -5,6 +5,14 @@ import * as util from "../util";
 const expectUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch(/[^.]unpack\(/);
 const expectTableUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("table.unpack");
 
+const spreadCases = [
+    "1, 2, ...[3, 4, 5]",
+    "...[1, 2], 3, 4, 5",
+    "1, 2, ...'spread', 4, 5",
```

Comment:
Should string spread be there? I'd rather test it as a part of `of string literal` (it also could be added to `in function call`)